src/types/context.ts

```diff
@@ -18,11 +18,11 @@ export interface Context<T extends SupportedEventsU = SupportedEventsU, TU exten
   config: PluginSettings;
   env: Env;
   logger: {
```

Comment:
```suggestion
  logger: Logs
```
I think you should get some inspiration from this other plugin:
https://github.com/ubiquibot/command-start-stop/blob/8017e89c0e9c289e60dda6e807bcf2351d52d907/src/types/context.ts#L21